src/main/scala/SerialAdapter.scala

```diff
@@ -303,8 +309,14 @@ case class AXIMemOverSerialTLClockParams(
     }
   }
 }
+case class SerialTLROMParams(
+  address: BigInt = 0x20000,
+  size: Int = 0x10000,
```

Comment:
Can't you just infer the size based off the file / array being put into the ROM? That would minimize unused space.

Reply:
The size is like a "max size". This is the same way it is done for the on-chip rom.
Also, since this is a ROM, having size > actual ROM contents size does not cost anything.

Reply:
Yea I know. I have a similar issue with the original TLROM... I still think exposing a parameter that technically isn't necessary is useless. I won't die on this hill though.